<launch>
  <!-- YOLOv8 ONNX 检测器 -->
  <node name="yolov8_detector" pkg="yolov8_ros" type="detector_onnx.py" output="screen">
    <param name="model_path" value="$(find yolov8_ros)/models/best.onnx" />
    <param name="class_names" value="circle_blue,circle_green,circle_red,object_blue,object_green,object_red" />
    <param name="conf_thres" value="0.4" />
    <param name="iou_thres" value="0.5" />
    <param name="device" type="string" value="cpu" />

  </node>
</launch>
